rslib/src/stats/graphs/future_due.rs

```diff
@@ -31,10 +32,16 @@ impl GraphsContext {
             }
             have_backlog |= due_day < 0;
             *due_by_day.entry(due_day).or_default() += 1;
+            if matches!(c.queue, CardQueue::Review | CardQueue::DayLearn) {
+                daily_load += 1.0 / c.interval as f32;
```

Comment:
```suggestion
                daily_load += 1.0 / c.interval.max(1) as f32;
```

We don't want to crash if a card has an interval of 0 set (which is invalid, but could happen with manual edits)